functions

```diff
@@ -745,8 +745,13 @@ function set_systemd_override {
     local key="$1"
     local value="$2"
 
-    local sysconf="/etc/systemd/system.conf"
-    iniset -sudo "${sysconf}" "Manager" "$key" "$value"
+    #local sysconf="/etc/systemd/system.conf"
+    #iniset -sudo "${sysconf}" "Manager" "$key" "$value"
+    local sysconf_40="/usr/lib/systemd/system.conf.d/40-core-ulimit.conf"
+    iniset -sudo "${sysconf_40}" "Manager" "$key" "$value"
+    local sysconf_50="/usr/lib/systemd/system.conf.d/50-nfiles.conf"
+    iniset -sudo "${sysconf_50}" "Manager" "$key" "$value"
```

Comment:
These files do not exist on the clear install I am looking at, should these go into /etc?